features/git-extract/cherry_pick_conflict/without_remote_origin/with_open_changes.feature

```diff
@@ -0,0 +1,99 @@
+Feature: git extract: resolving conflicts between main branch and extracted commits (with open changes and without remote origin)
+
+  (see ../with_remote_origin/with_open_changes.feature)
+
+
+  Background:
+    Given I have a feature branch named "feature"
+    And my repo does not have a remote origin
+    And the following commits exist in my repository
+      | BRANCH  | LOCATION | MESSAGE         | FILE NAME        | FILE CONTENT     |
+      | main    | local    | main commit     | conflicting_file | main content     |
+      | feature | local    | feature commit  | feature_file     |                  |
+      |         |          | refactor commit | conflicting_file | refactor content |
+    And I am on the "feature" branch
+    And I have an uncommitted file with name: "uncommitted" and content: "stuff"
+    When I run `git extract refactor` with the last commit sha
+
+
+  @finishes-with-non-empty-stash
+  Scenario: result
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                                      |
+      | feature  | git stash -u                                 |
+      | feature  | git checkout main                            |
+      | main     | git checkout -b refactor main                |
+      | refactor | git cherry-pick <%= sha 'refactor commit' %> |
+    And I get the error
+      """
+      To abort, run "git extract --abort".
+      To continue after you have resolved the conflicts, run "git extract --continue".
+      """
+    And I end up on the "refactor" branch
+    And I don't have an uncommitted file with name: "uncommitted"
+    And my repo has a cherry-pick in progress
+
+
+  Scenario: aborting
+    When I run `git extract --abort`
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                 |
+      | refactor | git cherry-pick --abort |
+      | refactor | git checkout main       |
+      | main     | git branch -d refactor  |
+      | main     | git checkout feature    |
+      | feature  | git stash pop           |
+    And I end up on the "feature" branch
+    And I again have an uncommitted file with name: "uncommitted" and content: "stuff"
+    And there is no "refactor" branch
+    And I have the following commits
```

Comment:
I have recently added a step called [And I am left with my original commits](https://github.com/Originate/git-town/blob/master/features/step_definitions/commit_steps.rb#L21). I'm in the process of cleaning up the feature specs and applying it everywhere. Since the commit table here is exactly the same as the one that defines the commits in the `Background` part, you can use it here to DRY up this spec even further.